README.markdown

```diff
@@ -261,6 +261,10 @@ protoc --plugin=node_modules/ts-proto/protoc-gen-ts_proto ./batching.proto -I.
 
   This is also useful if you want "only types".
 
+- With `--ts_proto_opt=outputPartialMethods=false`, the `Message.fromPartial` method for working with partial JSON-coded data will not be output.
```

Comment:
Fwiw the `fromPartial` methods are not for partially code JSON-data, they are for converting objects / object literals that programmers might have typed-out / made that look "basically like the protobuf message" but don't have every little thing right (like the default values applied for primitives).

That's still fine that you don't want them, but maybe word this as "the `Message.fromPartial` methods for accepting partially-formed objects/object literals will not be output".